modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicSQLServerIndexOnlineTest.java

```diff
@@ -0,0 +1,682 @@
+                    Double.parseDouble(elevationMetadata.split(",")[0]),
+                    Double.parseDouble(reader.getMetadataValue("ELEVATION_DOMAIN_MINIMUM")),
+                    1E-6);
+            assertEquals(
+                    Double.parseDouble(elevationMetadata.split(",")[1]),
+                    Double.parseDouble(reader.getMetadataValue("ELEVATION_DOMAIN_MAXIMUM")),
+                    1E-6);
+
+            // limit yourself to reading just a bit of it
+            final ParameterValue<GridGeometry2D> gg =
+                    AbstractGridFormat.READ_GRIDGEOMETRY2D.createValue();
+            final GeneralEnvelope envelope = reader.getOriginalEnvelope();
+            final Dimension dim = new Dimension();
+            dim.setSize(
+                    reader.getOriginalGridRange().getSpan(0) / 2.0,
+                    reader.getOriginalGridRange().getSpan(1) / 2.0);
+            final Rectangle rasterArea = ((GridEnvelope2D) reader.getOriginalGridRange());
+            rasterArea.setSize(dim);
+            final GridEnvelope2D range = new GridEnvelope2D(rasterArea);
+            gg.setValue(new GridGeometry2D(range, envelope));
+
+            // use imageio with defined tiles
+            final ParameterValue<List> time = ImageMosaicFormat.TIME.createValue();
+            final List<Date> timeValues = new ArrayList<Date>();
+            final SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.sss'Z'");
+            sdf.setTimeZone(TimeZone.getTimeZone("GMT+0"));
+            Date date = sdf.parse("2008-10-31T00:00:00.000Z");
+            timeValues.add(date);
+            time.setValue(timeValues);
+
+            final ParameterValue<double[]> bkg = ImageMosaicFormat.BACKGROUND_VALUES.createValue();
+            bkg.setValue(new double[] {-9999.0});
+
+            final ParameterValue<Boolean> direct =
+                    ImageMosaicFormat.USE_JAI_IMAGEREAD.createValue();
+            direct.setValue(false);
+
+            final ParameterValue<List> elevation = ImageMosaicFormat.ELEVATION.createValue();
+            elevation.setValue(Arrays.asList(100.0));
+
+            // Test the output coverage
+            assertNotNull(
+                    reader.read(new GeneralParameterValue[] {gg, time, bkg, elevation, direct}));
+            TestUtils.checkCoverage(
+                    reader,
+                    new GeneralParameterValue[] {gg, time, bkg, elevation, direct},
+                    "Time-Elevation Test");
+
+            // Test the output coverage
+            reader = TestUtils.getReader(timeElevURL, format);
+            elevation.setValue(Arrays.asList(NumberRange.create(0.0, 10.0)));
+            TestUtils.checkCoverage(
+                    reader,
+                    new GeneralParameterValue[] {gg, time, bkg, elevation, direct},
+                    "Time-Elevation Test");
+        } finally {
+            // dropTables(new String[] {tempFolderName1, "GEOMETRY_METADATA"});
```

Comment:
is it intentionally commented out?
needed for other type of testing?
the other tests have that line not commented.

Reply:
fixed